code/modules/events/spacevine.dm

```diff
@@ -469,7 +466,7 @@
 /obj/structure/spacevine/attack_alien(mob/living/user)
 	eat(user)
 
-/obj/structure/spacevine_controller
+/obj/effect/spacevine_controller
```

Comment:
The vine controller stays an effect, since it's indestructible and shouldn't be attackable.